x/storage/types/crosschain.go

```diff
@@ -562,7 +562,7 @@ func (p CreateBucketSynPackage) ValidateBasic() error {
 		ChargedReadQuota: p.ChargedReadQuota,
 	}
 
-	return msg.ValidateBasic()
+	return msg.ValidateRuntime(ctx)
```

Comment:
you can do `ValidateBasic` and `ValidateRuntime` independently

Reply:
Done